Formula/qt.rb

```diff
@@ -254,6 +255,18 @@ def install
     end
   end
 
+  def post_install
+    return if OS.mac?
+
+    # Ensure the hard-coded versioned `gcc` reference does not go stale.
+    ohai "Fixing up GCC references..."
+    gcc_version = Formula["gcc"].any_installed_version.major
+    inreplace(share/"qtbase/mkspecs/common/g++-base.conf") do |s|
```

Comment:
This path is still wrong.

Reply:
Maybe:
```suggestion
    inreplace(pkgshare/"mkspecs/common/g++-base.conf") do |s|
```

We do pass in `-DINSTALL_MKSPECSDIR=share/qt/mkspecs`

Reply:
I was pretty sure I checked the bottle for the right path though. Must've read it wrong.